tests/tacacs/utils.py

```diff
@@ -67,8 +67,8 @@ def log_exist(ptfhost, sed_command):
 
 
 def get_auditd_config_reload_timestamp(duthost):
```

Comment:
>get_auditd_config_reload_timestamp [](http://example.com/codeflow?start=4&length=34)

Could you explain in PR description why "new auditd rules add by auditd container break get_auditd_config_reload_timestamp method" ?

Reply:
After add new rules, the auditd restart take longer time, so 'sudo journalctl -u auditd --boot --no-pager' some time does not contains any 'audisp-tacplus re-initializing configuration' in result, which cause grep command failed